ext/net/lib.deno_net.d.ts

```diff
@@ -49,7 +49,12 @@ declare namespace Deno {
     /** The port to listen on. */
     port: number;
     /** A literal IP address or host name that can be resolved to an IP address.
-     * If not specified, defaults to `0.0.0.0`. */
+     * If not specified, defaults to `0.0.0.0`.
+     *
+     * __Note about 0.0.0.0__ While listeninig 0.0.0.0 works on all platforms,
```

Comment:
`listeninig` -> `listening`

```suggestion
     * __Note about 0.0.0.0__ While listening 0.0.0.0 works on all platforms,
```

Reply:
Thanks!